src/status_im/utils/ethereum/eip55.cljs

```diff
@@ -0,0 +1,22 @@
+(ns status-im.utils.ethereum.eip55
+  "Utility function related to [EIP55](https://eips.ethereum.org/EIPS/eip-55)
+
+   This EIP standardize how ethereum addresses should be printed as strings to validate checksum.
+
+   e.g. 0x5aAeb6053F3E94C9b9A09f33669435E7Ef1BeAed"
+  (:require [clojure.string :as string]
+            [status-im.utils.ethereum.core :as ethereum]))
+
+(defn verifyAddressChecksum [address]
+  "verify address checksum according to EIP 55"
+  (def adHash (ethereum/naked-address (ethereum/sha3
```

Comment:
It is more idiomatic to use a `let` to create local vars.

Reply:
updated